site/sass/components/_featured-content.scss

```diff
@@ -2,21 +2,55 @@
 // For more information about --shape-circle--size, --shape-circle--transform and --shape-color check background.scss
 
 .featured-content {
-  align-items: center;
-  display: flex;
-  flex-wrap: wrap;
   position: relative;
+  width: 100%;
+
+  @include mq($bkpt-articles--2-cols) {
+    height: 40vh;
+    min-height: 30rem;
```

Comment:
This styling feels to me like it's intrinsic to the image as opposed to the container (it's ultimately controlling the size of the image, from what I can gather). Consider moving it to the image, and migrating some of the min and max height stuff to `clamp` so we have more of a continuous sizing for the image based on available viewport height (or width!)

Reply:
Done.